app/models/spree/user_decorator.rb

```diff
@@ -18,10 +18,17 @@
   accepts_nested_attributes_for :ship_address
 
   attr_accessible :enterprise_ids, :enterprise_roles_attributes, :enterprise_limit, :bill_address_attributes, :ship_address_attributes
-  after_create :send_signup_confirmation
 
   validate :limit_owned_enterprises
 
+  # We use the same options as Spree and add :confirmable
+  devise :database_authenticatable, :token_authenticatable, :registerable, :recoverable,
+    :rememberable, :trackable, :validatable, :confirmable, :encryptable,
+    :reconfirmable => true, confirmation_keys: [ :id, :email ], :encryptor => 'authlogic_sha512'
```

Comment:
For instance https://github.com/spree/spree_auth_devise/blob/2-4-stable/app/models/spree/user.rb#L8

maybe we can just add `devise :confirmable`, would that make sense?